packages/plugin-di/src/factories/plugin.ts

```
@@ -0,0 +1,106 @@
+import type { PluginFactory, PluginOptions } from "../types";
+
+/**
+ * Get an instance from the container
+ * @param ctx
+ * @param item
+ * @param type
+ * @returns
+ */
+async function getInstanceFromContainer<T>(
+    ctx: interfaces.Context,
+    item: T | Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)